gocov-html.go

```diff
@@ -22,7 +22,7 @@ package main
 
 import (
 	"flag"
-	"github.com/matm/gocov-html/cov"
+	"github.com/Blackmage89/gocov-html/cov"
```

Comment:
Package name changed to `github.com/Blackmage89/gocov-html/cov` from `github.com/matm/gocov-html/cov`.